build/config/rmi_pacta_2023q4_FAKEpa2024ch.json

```
@@ -0,0 +1,29 @@
+{
+  "data_share_path": "2023Q4_20240424T120055Z",
```

```suggestion
  "data_share_path": "2023Q4_20240709T111731Z",
```

Relates to https://github.com/RMI-PACTA/pacta.data.preparation/pull/32